docker.go

```diff
@@ -457,7 +457,11 @@ func getImageSummary(imageName string) *types.ImageSummary {
 	filters := filters.NewArgs()
 	filters.Add("reference", imageName)
 	images, err := cli.ImageList(ctx, types.ImageListOptions{Filters: filters})
-	if err != nil || len(images) != 1 {
+	if err != nil {
+		log.Errorf("unable to retrieve image summary of %s: %s", imageName, err.Error())
```

Comment:
You should return something here. You could return `nil` to be consistent with the old code or return `err` but that may have some fun ramifications.